mathics/core/symbols.py

```diff
@@ -424,12 +424,9 @@ def __str__(self) -> str:
         return self.name
 
     def atom_to_boxes(self, f, evaluation) -> "String":
```

Comment:
The return type "String" -> "_BoxedString"

Reply:
Done